src/calculateRentalCost.js

```diff
@@ -4,7 +4,18 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const value = 40;
+  const minimumDiscount = 20;
+  const maximumDiscount = 50;
+  const basePrice = days * value;
+
+  if (days <= 2) {
+    return basePrice;
```

Comment:
The condition 'days <= 2' can be simplified to 'days < 3' since you are already checking for 'days >= 3' in the next condition.